js/piwik.js

```diff
@@ -3267,6 +3267,8 @@ if (typeof window.Piwik !== 'object') {
                 return cookieMatch ? decodeWrapper(cookieMatch[2]) : 0;
             }
 
+            configHasConsent = !getCookie('consent_removed');
```

Comment:
I see... I wonder if we maybe need to call `getCookie()`also after each `setCookieDomain()` and `setCookieNamePrefix()` and `setCookiePath()` and `setSiteId()` etc as the domain on which the cookie is set might change?

I also wonder if we in general need to do `getCookie(getCookieName('consent_removed'))` and `setCookie(getCookieName('consent_removed')` and the same for `consent` etc